treniformis/README.md

```diff
@@ -0,0 +1,11 @@
+# vessel-lists
```

Comment:
@bitsofbits The readme needs to be translated to RST so it renders at PyPi, which does not support Markdown.